docker-compose.yml

```diff
@@ -166,7 +166,7 @@ services:
 
   preprints:
     image: quay.io/centerforopenscience/ember-preprints:develop
-    command: /bin/bash -c "npm install && ./node_modules/bower/bin/bower install --allow-root --config.intedractive=false && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4200"
+    command: /bin/bash -c "yarn --pure-lockfile --no-progress --no-emoji && ./node_modules/bower/bin/bower install --allow-root --config.interactive=false && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4200"
```

Comment:
This is a change to the docker-compose file, so It's using the preprints image, but running a different command. This is so that, when developing, the local changes will be synced in the container, and ember serve will be running (which will trigger ember builds as files are changed).

The reason for the extra flags `--no-progress` and `--no-emoji` is that console output console output is superfluous here. You want the output during a regular image build (`docker built --tag whatever .`). Yarn is smart enough to know when it's in a CI environment, so it's not necessary to add these flags in the travis config.